docs/data/README.mdx

```diff
@@ -2,10 +2,27 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+Stellar supports tools that facilitate communication between users and applications on the Stellar network. Users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+- **RPC** - live network gateway
+- **Horizon** - API for network state data
+- **Galexie** - exports raw ledger metadata files
+- **Hubble** - OLAP database for network data
+
+| Features                | RPC | Horizon | Galexie | Hubble |
+| ----------------------- | --- | ------- | ------- | ------ |
+| Real-time Data          | ✅  | ✅      | ✅      | ❌     |
+| Historical Data         | ❌  | ❌      | ✅      | ✅     |
+| Assets Data             | ❌  | ✅      | ✅      | ✅     |
+| Smart Contracts         | ✅  | ❌      | ✅      | ✅     |
+| Stellar Asset Contracts | ✅  | ✅      | ✅      | ✅     |
+| API                     | ✅  | ✅      | ❌      | ❌     |
+| Transaction Submission  | ✅  | ✅      | ❌      | ❌     |
+| Curated and Parsed Data | ✅  | ✅      | ❌      | ✅     |
```

Comment:
oooh maybe unpopular opinion, but I would put ❌ for RPC for curated/parsed data

Reply:
Ooo that is unpopular. I'm fine with doing that just for the sake of differentiating RPC and Horizon more

Reply:
updated in e250dd6664b5c0a62fb1d27846ac2633cd0ae9f7

Reply:
:shocked-pikachu-face: